scripts/setup_gemdaq.sh

```diff
@@ -190,15 +270,16 @@ then
         then
             if [ -z "$VFATQC_DEV_VERSION" ]
             then
-                echo wget https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
-                wget https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
+                echo $WGET https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
+                $WGET https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
             else
-                echo wget https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}-dev${VFATQC_DEV_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
-                wget https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}-dev${VFATQC_DEV_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
+                echo $WGET https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}-dev${VFATQC_DEV_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
+                $WGET https://github.com/cms-gem-daq-project/vfatqc-python-scripts/releases/download/v${VFATQC_VERSION}-dev${VFATQC_DEV_VERSION}/gempython_vfatqc-${VFATQC_VERSION}.tar.gz
             fi
         fi
-    echo python -m pip install gempython_vfatqc-${VFATQC_VERSION}.tar.gz
-    python -m pip install gempython_vfatqc-${VFATQC_VERSION}.tar.gz
+        ls >/dev/null # Forces a filesystem sync
```

Comment:
My own ignorance here, but what's this line for?

Reply:
At P5, since `wget` is executed remotely on `cmsusr`, the file appears on the local machine after some delay, and hence the installation fails. For whatever reason, `ls` appears to cause a synchronization, and the file is present afterwards. I'd be happy to find a cleaner solution.

Reply:
If you can find a cleaner solution sure, but otherwise no problem.  Yes I've noticed the same behavior.

Reply:
If you just need to introduce a delay, use a `sleep` command. E.g. if you want to pause for 0.5 second, you need `sleep 0.5`
Same comment in every place where you have `ls >/dev/null`

Reply:
Tried `sleep 1`, didn't work. `ls` is instant and works.

Reply:
What means it doesn't work? throws an error? or what?

Reply:
> What means it doesn't work? throws an error? or what?

The downloaded file doesn't appear on the local machine within 1s.